pyaedt/hfss.py

```diff
@@ -696,7 +718,7 @@ def assign_coating(
             props["IsShellElement"] = issheelElement
         else:
             props["IsInternal"] = isInternal
-        return self._create_boundary("Coating_" + listobjname[:32], props, "Finite Conductivity")
+        return self._create_boundary("Coating_" + listobjname[1:32], props, "Finite Conductivity")
```

Comment:
For my knowledge why is it `[1:32]`? Why 32?

Reply:
@MaxJPRey that's a good question, I think it is a legacy convention to have less than 32 characters, @maxcapodi78 @Alberto-DM do you know if the limit of 32 is cosmetic or not?